### PR TITLE
Improves excerpt algorithm, removes deprecated DocumentHeader

### PR DESCRIPTION
**Pull request by vanny96**

### CODE REVIEW

File: app/models/Post.scala [hunk 1]
```diff
@@ -15,23 +16,27 @@ object Post {
...
+
   lazy val htmlContent: String = Post.asciidoctor.convert(
     content,
-    new java.util.HashMap[String, Object]())
-
-  val header: DocumentHeader = Post.asciidoctor.readDocumentHeader(content)
+    Options.builder().build())
 
-  lazy val authorName: String = header.getAuthor.getFullName
+  lazy val authorName: String = document.getAuthors.get(0).getFullName
```
Comment (wjglerum) [inline app/models/Post.scala:25]: There could potentially be more authors (in the future). What do you think about joining them together?

```suggestion
  lazy val authorName: String = document.getAuthors.asScala.map(_.getFullName).mkString(", ")
```

File: app/models/Post.scala [hunk 2]
```diff
@@ -1,11 +1,12 @@
 package models
 
-import org.asciidoctor.Asciidoctor
+import org.asciidoctor.{Asciidoctor, Options}
 import org.asciidoctor.Asciidoctor.Factory
-import org.asciidoctor.ast.DocumentHeader
+import org.asciidoctor.ast.{Document, DocumentHeader}
```
Comment (wjglerum) [inline app/models/Post.scala:5]: You can now cleanup the deprecated import too
```suggestion
import org.asciidoctor.ast.Document
```

**Review by wjglerum — COMMENTED** [review]
Good idea to get rid of those deprecated methods!

File: app/controllers/HomeController.scala [hunk 3]
```diff
@@ -119,7 +119,7 @@ class HomeController @Inject()(
     request.get().flatMap { r => {
       val post = Post(s"/${name}", s"https://raw.githubusercontent.com/${organization}/${repository}/${branch}/media/${name}/background.png", r.body)
       // TODO Change me
```
Comment (azthec) [inline app/controllers/HomeController.scala:121]: Nit: guess we can clean this up

File: app/models/Post.scala (hunk 1 above)
Comment (wjglerum) [reply app/models/Post.scala:25]: Hmm, looking at the rest of the code that would actually break some other things. 

So we might not want to do that (yet)...

Comment (vanny96) [reply app/models/Post.scala:26]: Introduced authorsNames as suggested by Rui!

**Review by wjglerum — APPROVED** [review]

**Pull request merged**